tests/python/test_cast.py

```diff
@@ -39,3 +39,23 @@ def func():
         ret[None] = t
 
     func()
+
+
+@ti.all_archs
+def _test_bit_cast():
```

Comment:
Please name it `test_...` without `_` at the beginning if you want to trigger this test.

Reply:
+1. I tested this locally and it failed, seems like `bit_cast` isn't defined in the python scope?